lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -309,17 +310,69 @@ def compile_mixins(constant)
               indented("extend(#{qualified_name_of(mod)})")
             end
 
-          mixes_class_methods = extend
-            .select do |mod|
-              qualified_name_of(mod) == "::ActiveSupport::Concern" &&
-                Module === resolve_constant("#{name_of(constant)}::ClassMethods")
+          (prepends + includes + extends).join("\n")
+        end
+
+        sig { params(constant: Module).returns(String) }
+        def compile_mixes_in_class_methods(constant)
+          return "" if constant.is_a?(Class)
+
+          mixins_from_modules = {}
+
+          Class.new do
+            def method_missing(symbol, *args)
+              super
```

Comment:
Isn't support going to raise an error?

Reply:
Indeed, I was mislead by the Rubocop rule, this is fixed.

Reply:
As for `method_missing`, the reason is what I noted in the PR description:

> ... defines `method_missing` both on the instance and class level so that we can ignore calls to some expected methods on the including class coming from some modules as they are being includes/extended. For example, `IdentityCache` just plain assumes that it will be able to call some AR methods in the `self.included` method. Implementing an empty `method_missing` allows those calls to happen in a no-op manner.